packages/adapters/bitcoin/src/adapter.ts

```diff
@@ -256,6 +256,11 @@ export class BitcoinAdapter extends AdapterBlueprint<BitcoinConnector> {
     return Promise.resolve('0x')
   }
 
+  public override async walletGetAssets(
+    _params: AdapterBlueprint.WalletGetAssetsParams
```

Comment:
```suggestion
    params: AdapterBlueprint.WalletGetAssetsParams
```

Tiny but just to keep it same across adapters

Reply:
@enesozturk  for all the methods in this adapter where the params are not used inside the method it is marked with `_params` to avoid eslint:error 

```params' is defined but never used. Allowed unused args must match /^_/u```